fastddsspy_participants/src/cpp/model/DataStreamer.cpp

```diff
@@ -42,14 +42,15 @@ bool DataStreamer::activate_all(
 }
 
 bool DataStreamer::activate(
-        const ddspipe::core::types::DdsTopic& topic_to_activate,
+        const ddspipe::core::types::WildcardDdsFilterTopic& topic_to_activate,
+        const std::set<eprosima::ddspipe::core::types::DdsTopic>& topics,
         const std::shared_ptr<CallbackType>& callback)
 {
-    if (!is_topic_type_discovered(topic_to_activate))
+    if (!is_any_topic_type_discovered(topics))
```

Comment:
I believe it would be better to have this in the tool (Controller.cpp), it seems a bit weird to pass the topics like that.

Reply:
I do not understand your comment about this

Reply:
This check used to be done both in the controller (Controller::print_command_), and then again here. I asked Lucía to keep it only in one place, and she left it here, but I feel it would be better only in the controller. Two reasons:
- The behaviour it's the same but the way it is now is more restrictive, and I think it's better to let the user of this class (controller in this case) to set the restriction.
- To avoid having to pass topics as argument, which seems a bit odd to me.

Reply:
Not yet applied.

Reply:
I agree